azure-devops-extension/run-automator-build-task/package.json

```diff
@@ -16,46 +16,49 @@
   },
   "author": "Andrei Bitca, Daniel Mon Chit & Lukas Kalcok",
   "license": "ISC",
+  "resolutions": {
+    "cross-fetch": "3.1.5",
+    "minimatch": "7.4.2",
+    "ws": "8.12.1"
+  },
   "devDependencies": {
     "@types/jest": "^26.0.22",
-    "@types/nock": "^11.1.0",
-    "@types/node-fetch": "^2.5.7",
-    "@types/pino": "^6.3.2",
+    "@types/node-fetch": "^2.6.2",
     "@types/q": "^1.5.4",
     "@types/sinon": "^9.0.8",
-    "@typescript-eslint/eslint-plugin": "^4.1.1",
-    "@zerollup/ts-transform-paths": "^1.7.18",
-    "eslint": "^7.9.0",
-    "eslint-config-deepcrawl": "^5.6.0",
+    "@typescript-eslint/eslint-plugin": "^5.54.1",
+    "eslint": "^8.35.0",
+    "eslint-config-deepcrawl": "^8.1.0",
```

Comment:
After upgrade you should remove unnecessary dependencies, we simplified config to include plugins: https://github.com/deepcrawl/eslint-config/blob/2f855b7749948531a73c6624f64babdeaf5db256/package.json#L23

Reply:
Redundant eslint plugin dependencies removed